.github/workflows/weekly.yml

```diff
@@ -0,0 +1,277 @@
+            exit 1
+          fi
+      - name: Upload Inductor XPU E2E Data
+        if: ${{ ! cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: Inductor-XPU-E2E-Data-${{ github.event.pull_request.number || github.sha }}
+          path: ${{ github.workspace }}/upload_files
+
+  Tests-Failure-And-Report:
+    if: ${{ ! cancelled() }}
+    runs-on: pvc_e2e
+    permissions:
+      issues: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+      python: ${{ github.event_name == 'schedule' && '3.10' || inputs.python }}
+    needs: Linux-Weekly-Ondemand-E2E-Tests
+    steps:
+      - name: Report github issue for XPU OPS Weekly
+        if: github.repository_owner == 'intel'
+        run: |
+          set -xe
+          # Test env
+          build_url="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          repo="${{ github.repository }}"
+          TORCH_BRANCH_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TORCH_BRANCH_ID }}"
+          TORCH_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TORCH_COMMIT_ID }}"
+          DRIVER_VERSION="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.DRIVER_VERSION }}"
+          BUNDLE_VERSION="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.BUNDLE_VERSION }}"
+          OS_PRETTY_NAME="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.OS_PRETTY_NAME }}"
+          GCC_VERSION="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.GCC_VERSION }}"
+          TORCHBENCH_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TORCHBENCH_COMMIT_ID }}"
+          TORCHVISION_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TORCHVISION_COMMIT_ID }}"
+          TORCHAUDIO_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TORCHAUDIO_COMMIT_ID }}"
+          TRANSFORMERS_VERSION="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TRANSFORMERS_VERSION }}"
+          TIMM_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TIMM_COMMIT_ID }}"
+          TRITON_COMMIT_ID="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TRITON_COMMIT_ID }}"
+          TIMEOUT_MODELS="${{ needs.Linux-Weekly-Ondemand-E2E-Tests.outputs.TIMEOUT_MODELS }}"
+          # Test status
+          if [ "${{ needs.Linux-Weekly-Ondemand-E2E-Tests.result }}" == "success" ];then
+            test_status=Success
+          elif [ "${{ needs.Linux-Weekly-Ondemand-E2E-Tests.result }}" == "failure" ];then
+            test_status=Failure
+            cc_comment="CC ${{ secrets.Weekly_EMAIL_LIST }}"
+          else
+            test_status=None
+            exit 0
+          fi
+          # Test Type
+          if [ "${GITHUB_EVENT_NAME}" == "workflow_dispatch" ];then
+            test_type="On-demand"
+            test_issue_id=426
+            cc_comment="CC @${GITHUB_TRIGGERING_ACTOR}"
+          else
+            test_type="Weekly"
+            test_issue_id=432
```

Comment:
Do we need a new issue for weekly?

Reply:
I think not, just test scope is different, and we can cancel the nightly in weekend, is it okay?